manifests/init.pp

```diff
@@ -109,136 +109,47 @@
   Boolean $manage_broker = $foreman_proxy_content::params::manage_broker,
 ) inherits foreman_proxy_content::params {
   include ::certs
+  include ::certs::foreman_proxy
   include ::foreman_proxy
   include ::foreman_proxy::plugin::pulp
+  Class['certs::foreman_proxy'] ~> Class['foreman_proxy::service']
 
-  $pulp = $::foreman_proxy::plugin::pulp::pulpnode_enabled
+  $pulp_node = $::foreman_proxy::plugin::pulp::pulpnode_enabled
```

Comment:
We really should replace the node nomenclature.

Reply:
Indeed, but what should be the correct name?

Reply:
I reached out to pulp-dev list to look for commonly accepted nomenclature. The best I could arrive at is 'parent' and 'child' from the responses I got.

Reply:
Given in how many places this naming is currently used, I'm leaning to keeping that outside of this refactor.